apps/console/src/features/roles/components/edit-role/edit-role-permission.tsx

```diff
@@ -162,9 +162,17 @@ export const UpdatedRolePermissionDetails: FunctionComponent<RolePermissionDetai
                 if (
                     !selectedAPIResources.find((selectedAPIResource: APIResourceInterface) =>
                         selectedAPIResource?.id === api?.id)) {
+                    // Remove this once the backend improvement is done to send the type of the API resource.
+                    let type: string = APIResourcesConstants.SYSTEM;
+
+                    if (api?.identifier?.startsWith("/o/")) {
+                        type = APIResourcesConstants.SYSTEM_ORG;
+                    }
```

Comment:
Let's use a ternary here and make the variable a constant.  

Also let's use a more meaningful variable name.

Fix the other places as well.